charts/argo-cd/Chart.yaml

```diff
@@ -23,4 +23,5 @@ dependencies:
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
+    - "[Added]: Extra secret labels with .Values.configs.secret.labels"
     - "[Changed]: Update Argo CD extensions to v0.2.1"
```

Comment:
This PR is not updating Argo CD extensions

Reply:
Should I add some changelog then?